src/development/troubleshoot.md

```diff
@@ -74,6 +74,35 @@ The memory usage of your container exceeds the limit, the kernel thus kills the
 * Use [sizing hints](https://docs.platform.sh/languages/php.html#php-worker-sizing-hints) to reduce the amount of PHP workers which reduces the memory footprint.
 * Upgrade your subscription on Platform.sh to get more computing resources. To do so, log into your [account](https://accounts.platform.sh) and edit the project.
 
+## Low disk space
+
+If you suspect you are running low on disk space on your application container, the easiest way to check it is to login using `platform ssh` and run the `df` command.  `df` has numerous options to tweak its output, but for just checking the available writeable space the simplest option is: `df -h -x tmpfs -x squashfs | grep -v /run/shared`
```

Comment:
If you add `-a` you'll get to see all the mounts, not just the first on each volume, but then you have to filter out more:

```sh
df -a -h -x proc -x squashfs -x tmpfs -x devpts -x sysfs | grep -v /run/shared
```

Might seem a bit ridiculous describing that as the "simplest option" (when `df` theoretically shows the necessary % on its own).

Reply:
Perhaps change it to "most direct"?  My first version used `df -haT` to get all possible data, but as @sruggier pointed out that's TMI for most people who just want to know how much disk mount  space they are using.